devlib/platform/gem5.py

```diff
@@ -174,7 +174,11 @@ def _intercept_telnet_port(self):
             # Open the stderr file
             with open(self.stderr_filename, 'r') as f:
                 for line in f:
+                    # Look for two different strings, exact wording depends on
+                    # version of gem5
                     m = re.search(r"Listening for system connection on port (?P<port>\d+)", line)
+                    if not m:
+                        m = re.search(r"Listening for connections on port (?P<port>\d+)", line)
```

Comment:
Could condense these into a single expression, e.g. something like:
`Listening for (system )?connection[s]? on port (?P<port>\d+)`

Reply:
To be fair, I like splitting it up in 2 lines, because then it is more readable, if you would be okay with that?